projects/angular-sdk/src/lib/cinchy.service.ts

```diff
@@ -119,7 +119,16 @@ export class CinchyService implements OnDestroy {
                     // We want to make sure that we only modify the URL after it has done its round trip, otherwise there's
                     // a risk that it will either be mangled or that it could cause a mismatch between the URL and the list
                     // of allowed URLs in the app's security policy'
-                    const routerSubscription = that._router.events.subscribe({
+                    const routerSubscription = that._router.events
```

Comment:
This change is necessary to accommodate the stricter typing of the upgraded router.

https://angular.io/api/router/RouterEvent